src/Eleventy.js

```diff
@@ -778,20 +778,15 @@ Arguments:
 
 	shouldTriggerConfigReset(changedFiles) {
 		let configFilePaths = new Set(this.eleventyConfig.getLocalProjectConfigFiles());
-		for (let filePath of changedFiles) {
-			if (configFilePaths.has(filePath)) {
-				return true;
-			}
+		if (changedFiles.intersection(configFilePaths).size > 0) {
```

Comment:
Forgive my ignorance but is this really supported in Node yet?

The following works in my Chrome browser console, but not in the latest version of Node.js (v21.7.2).

```
new Set([1, 2, 3]).intersection(new Set([2, 3, 4]))
```

![Screenshot 2024-04-05 at 17 05 52 (Ghostty)](https://github.com/11ty/eleventy/assets/47499684/7b691f68-df9d-4dd0-b81e-c10ad2485a95)
![Screenshot 2024-04-05 at 17 06 47 (Arc)](https://github.com/11ty/eleventy/assets/47499684/ed7290b1-a2cb-4cd4-a440-1a797710efb4)

Reply:
node.green has this only in Nightlies so far. We need all code to be Node 18+ for the 3.0 release.

Reply:
Aw, dang! **Still** no set operations in JS? 🤦‍♂️ 

Come on, Node.js devs…


(I’ll have this fixed shortly!)